.vscode/settings.json

```
@@ -98,7 +97,7 @@
     "pwsh": null
   },
 
-  "terminal.integrated.defaultProfile.linux": "fish",
+  // "terminal.integrated.defaultProfile.linux": "fish",
```

```suggestion
  // "terminal.integrated.defaultProfile.linux": "bash",
```

this part might be relevant to postCreate script that runs (thinking about entrypoints)